src/python/pants/notes/master.rst

```diff
@@ -4,6 +4,161 @@ Master Pre-Releases
 This document describes development releases which occur weekly from master, and which have
 not undergone the vetting associated with ``stable`` releases.
 
+1.28.0.dev1 (5/1/2020)
+-----------------------
+
+API Changes
+~~~~~~~~~~~
+
+* Replace `list` with `list-v2` when in V2 exclusive mode (#9671)
+  `PR #9671 <https://github.com/pantsbuild/pants/pull/9671>`_
```

Comment:
Technically an API change, but I think put this in Refactoring. It only users in V2 exclusive mode, which is almost entirely brand new users.